src/Symfony/Component/HttpKernel/Tests/Fixtures/Controller/ExtendingSession.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
```

Comment:
Small somewhat unrelated sidenote, I noticed only the `NullableController.php` has a copyright docblock but the other do not... Most of the fixture files have them, should it also be added?